README.md

```diff
@@ -126,7 +128,7 @@ TF_LOG_PROVIDER=INFO terraform refresh
 TF_LOG_PROVIDER=INFO terraform show
 ```
 
-### Stop Task
+### End Task
```

Comment:
```suggestion
### Delete Task
```

More accurate?

Reply:
- stop: clashes with "stop instance" meaning
- delete: only applies to storage data, not instance
- terminate: only applies to instance, not storage data
- destroy: accurate Terraform term, but sounds weird and also is a self-referential definition?
- finish/end: new term

I'd say finish/end/destroy in order of preference

Reply:
> _delete_

Delete like in “deleting a cloud resource” and not like in “deleting a file”. Niche, but accurate enough. ⚔️

Reply:
> _destroy_

Terraform–specific and a bit exagerated, but also pretty descriptive.

Reply:
> _cleanup_

Reminder: it also downloads data

Reply:
For the sake of conceptual creativity: the original XPD proposal used “harvest”

Reply:
leaving this unresolved until someone comes up with any better ideas :)

Reply:
👍🏼